File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/DeleteApplicationVersionExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class DeleteApplicationVersionExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            new ApplicationsBetaApi(config).DeleteApplicationVersion(
                applicationKey: "applicationKey_string",
                versionKey: "versionKey_string"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling ApplicationsBetaApi#DeleteApplicationVersion: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/DeleteApplicationVersionExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.ApplicationsBetaApi(configuration).deleteApplicationVersion(
  "applicationKey_string", // applicationKey
  "versionKey_string", // versionKey
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling ApplicationsBetaApi#deleteApplicationVersion:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
